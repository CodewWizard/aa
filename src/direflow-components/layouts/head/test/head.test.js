import React from "react";
import renderer from "react-test-renderer";
import PageHead from "../head";

describe("PageHead", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <PageHead
          title="Test Title"
          description="Test Description"
          favicon="favicon.ico"
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
