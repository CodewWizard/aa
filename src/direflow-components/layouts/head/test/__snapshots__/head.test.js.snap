// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHead renders correctly 1`] = `
<a>
  <title>
    Test Title
  </title>
  <meta
    content="Test Description"
    name="description"
  />
  <meta
    content="minimum-scale=1, initial-scale=1, width=device-width"
    name="viewport"
  />
  <meta
    content="no-cache"
    http-equiv="cache-control"
  />
  <meta
    content="0"
    http-equiv="expires"
  />
  <meta
    content="no-cache"
    http-equiv="pragma"
  />
  <link
    href="favicon.ico"
    rel="icon"
  />
</a>
`;
